Open and transparent examples (#1515)
sourcegraph/about: company/values.md

1. Share your document in an appropriate channel with the same context that you added at the top of the document.

> My team has goals and plans for the future, but I am hesitant to put this in the handbook because I don't want to promise anything to our customers and I don't want competitors copying us.

We document our goals and plans publicly so we can engage with customers to build the best product possible. [Documenting a goal is not the same thing as committing to a customer](goals/guidelines.md#choosing-goals). If our competitors are closely following what we are doing, then we are clearly doing a good job and we should keep doing what we are doing!

> I am working on improving the security of our product, but I feel uncomfortable clearly documenting what I am doing (e.g., in public status updates, issues, PR descriptions, the changelog) because I am worried that it will make Sourcegraph look bad.

> Our code is public and we do our development in the public, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.

Reviewer: ```suggestion
Our code is public and we do our development in the open, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.
```

Sounds more natural to me, mostly because of using the word public twice so close together.

Author: ```suggestion
Our code and PRs are public, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.
```
